core/core/controller.py

```diff
@@ -6,8 +6,6 @@
 
 services_map = {
     # Base
-    Actions.HELLO.value: lambda _: Response(message=CommandMessages.HELP_QUESTION),
-    Actions.HELP.value: lambda _: None,
```

Comment:
I just realized that for the core module, there is no need for these actions.

Reply:
👌